LeaderboardBackend.Test/Leaderboards.cs

```diff
@@ -323,51 +323,70 @@ public async Task GetLeaderboards_OK()
         ApplicationContext context = _factory.Services.CreateScope().ServiceProvider.GetRequiredService<ApplicationContext>();
         await context.Leaderboards.ExecuteDeleteAsync();
 
+        Instant now = _clock.GetCurrentInstant();
+        _clock.AdvanceSeconds(1);
+        Instant later = _clock.GetCurrentInstant();
```

Comment:
CreatedAt will automatically be calculated on insertion so you don't need to set it explicitly when adding the record to the database.